CHANGELOG.md

```diff
@@ -54,6 +54,18 @@ Other Changes
 - Lepton was upgraded to the latest version (PR #349)
 - Made Object::print a const member function (PR #191)
 - Improved the testOptimization/OptimizationExample to reduce the runtime (PR #416)
+- Added shouldBeParallelIfPossible() override methods to all computationally
+expensive muscle force classes
```

Comment:
Thanks for the background. I believe `shouldBeParallelIfPossible` can be confusing. The individual force isn't being parallelized. What you want to relate is whether or not the dynamics of given force element can be computed in parallel with other forces, right? 

I am also wary about using this method to also indicate the force computation is not threadsafe. It will still be a problem if any other force says it wants to be computed in parallel, because it will be fired off to another thread. So you have to check that you don't have a single false flag in case that one is not thread-safe. If the force is not threadsafe (or unsafe?) it might be better to have a separate flag to indicate that knowledge. If a force is marked as not threadsafe, we could automatically apply a lock so that we still take advantage of parallelization of other forces?